deepchecks/tabular/suites/default_suites.py

```diff
@@ -325,6 +330,129 @@ def model_evaluation(alternative_scorers: Dict[str, Callable] = None,
+                     random_state: int = 42,
+                     n_to_show: int = 5,
+                     **kwargs) -> Suite:
+    """Suite for testing the model in production.
+
+    The suite contains checks for evaluating the model's performance. Checks for detecting drift and checks for data
+    integrity issues that may occur in production.
+
+    List of Checks (exact checks depend on the task type and the is_comparative flag):
+        .. list-table:: List of Checks
+           :widths: 50 50
+           :header-rows: 1
+
+           * - Check Example
+             - API Reference
+           * - :ref:`plot_tabular_roc_report`
+             - :class:`~deepchecks.tabular.checks.model_evaluation.RocReport`
+           * - :ref:`plot_tabular_confusion_matrix_report`
+             - :class:`~deepchecks.tabular.checks.model_evaluation.ConfusionMatrixReport`
+           * - :ref:`plot_tabular_weak_segment_performance`
+             - :class:`~deepchecks.tabular.checks.model_evaluation.WeakSegmentPerformance`
+           * - :ref:`plot_tabular_regression_error_distribution`
+             - :class:`~deepchecks.tabular.checks.model_evaluation.RegressionErrorDistribution`
+           * - :ref:`plot_tabular_string_mismatch_comparison`
+             - :class:`~deepchecks.tabular.checks.train_test_validation.StringMismatchComparison`
+           * - :ref:`plot_tabular_feature_label_correlation_change`
+             - :class:`~deepchecks.tabular.checks.train_test_validation.FeatureLabelCorrelationChange`
+           * - :ref:`plot_tabular_train_test_feature_drift`
+             - :class:`~deepchecks.tabular.checks.train_test_validation.TrainTestFeatureDrift`
+           * - :ref:`plot_tabular_train_test_label_drift`
+             - :class:`~deepchecks.tabular.checks.train_test_validation.TrainTestLabelDrift`
+           * - :ref:`plot_tabular_multivariate_drift`
+             - :class:`~deepchecks.tabular.checks.train_test_validation.MultivariateDrift`
+           * - :ref:`plot_tabular_train_test_prediction_drift`
+             - :class:`~deepchecks.tabular.checks.model_evaluation.TrainTestPredictionDrift`
+           * - :ref:`plot_tabular_train_test_prediction_drift`
+             - :class:`~deepchecks.tabular.checks.model_evaluation.TrainTestPredictionDrift`
+           * - :ref:`plot_tabular_string_mismatch`
+             - :class:`~deepchecks.tabular.checks.data_integrity.StringMismatch`
+           * - :ref:`plot_tabular_feature_label_correlation`
+             - :class:`~deepchecks.tabular.checks.data_integrity.FeatureLabelCorrelation`
+           * - :ref:`plot_tabular_feature_feature_correlation`
+             - :class:`~deepchecks.tabular.checks.data_integrity.FeatureFeatureCorrelation`
+           * - :ref:`plot_tabular_single_dataset_performance`
+             - :class:`~deepchecks.tabular.checks.model_evaluation.SingleDatasetPerformance`
+
+    Parameters
+    ----------
+    task_type : str
```

Comment:
Make this optional - if it is not supplied add all the checks (for regression and classification)

Reply:
Alternatively, remove this argument and always add both checks